nationalarchives/ds-wagtail
# [UN-549] Record Revealed - Missing icon

Ticket URL: https://national-archives.atlassian.net/browse/UN-549

## About these changes

Swaps existing icon include for an SVG sprite so that it can inherit the css colour correctly and show in high contrast mode.

## How to check these changes

1. View record revealed page locally
2. Open the images and transcripts component
3. Open devtools, click the 3 dots and open More Tools > Rendering:
![image](https://user-images.githubusercontent.com/8680557/234261633-39ecb7fc-2426-4b34-8dac-eb659d7003a3.png)

4. Change the forced-colors setting to 'forced-colors: active' to emulate high contrast mode - you can set this to 'light' or 'dark' theme by changing 'prefers-color-theme' setting as well:
![image](https://user-images.githubusercontent.com/8680557/234261728-e5e6609d-707b-48de-943c-67f6aad054fb.png)
![image](https://user-images.githubusercontent.com/8680557/234262146-b1ffe8f5-ea8d-4af3-810a-e6f6e595d4b8.png)


5. See that the close icon is still visible in high contrast mode 

![image](https://user-images.githubusercontent.com/8680557/234262241-5574c48c-5bbe-4946-8231-bb5592d0d2b7.png)
![image](https://user-images.githubusercontent.com/8680557/234262294-931bf2be-8752-4fce-ab52-899f060eb6ce.png)


## Before assigning to reviewer, please make sure you have


## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)
